presto-main/src/test/java/io/prestosql/execution/TestCommitTask.java

```diff
@@ -112,8 +112,7 @@ public void testUnknownTransactionCommit()
             getFutureValue(new CommitTask().execute(new Commit(), transactionManager, metadata, new AllowAllAccessControl(), stateMachine, emptyList()));
             fail();
         }
-        catch (PrestoException e) {
-            assertEquals(e.getErrorCode(), UNKNOWN_TRANSACTION.toErrorCode());
+        catch (NotInTransactionException expected) {
```

Comment:
Should `NotInTransactionException` be a `PrestoException` with `UNKNOWN_TRANSACTION`? 
Or `UNKNOWN_TRANSACTION` should be removed? (it's unused now)